o/opentelemetry-php-contrib/build_info.json

```diff
@@ -0,0 +1,20 @@
+{
+    "package_name" : "opentelemetry-php-contrib",
+    "github_url":"https://github.com/open-telemetry/opentelemetry-php-contrib",
+    "version": "1.0.0beta2",
+    "default_branch": "main",
+    "package_dir": "o/opentelemetry-php-contrib",
+    "validate_build_script": "true",
+    "build_script" : "opentelemetry-php-contrib_1.0.0beta2_ubi_8.5.sh",
+    "use_non_root_user": "true",
```

Comment:
@Pratikt2312 `"use_non_root_user": "true"` - this means your build script does not need any sudo level permission and can work with normal user. However, I see you have dnf install in build script so this wont work. Please revalidate build script with normal user and sudo user to correctly set this flag.

Reply:
Ok, I'll check and update.

Reply:
Revalidated the script and updated the changes.